src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FilterFunctions_V1CompatDisabled.txt

```diff
@@ -0,0 +1,7 @@
+﻿#SETUP: EnableExpressionChaining,MutationFunctionsTestSetup,disable:PowerFxV1CompatibilityRules
+
+>> With({t1:Table({a:1}), t2:Table({a:1})}, LookUp(t1, a=Collect(t2, {a:2}).a))
+Blank()
+
+>> With({t1:Table({a:1}), t2:Table({a:1})}, Filter(t1, a=Collect(t2, {a:2}).a))
+Table()
```

Comment:
We probably don't need this test - we don't want to specify that this behavior is "expected" - we just allow it for legacy reasons